include/swift/SIL/SILModule.h

```diff
@@ -178,7 +178,7 @@ class SILModule {
   llvm::DenseMap<const ClassDecl *, SILVTable *> VTableMap;
 
   /// The list of SILVTables in the module.
-  VTableListType vtables;
+  std::vector<SILVTable*> vtables;
```

Comment:
Maybe we could use an `llvm::MapVector` here, so that we get both stable order and efficient lookup?

Reply:
I just looked at the `llvm::MapVector` implementation. It's less time and space efficient than what this pull request is proposing (in short: "map of keys to indices into a vector of key/value pairs"). It's also less convenient because the `std::vector` is key/value pairs when all we need/want is an array of values.

Reply:
Alright, sounds good.